src/Jackett.Common/Definitions/yggtorrent.yml

```diff
@@ -6,7 +6,7 @@
   type: semi-private
   encoding: UTF-8
   links:
-    - https://www.yggtorrent.is/
```

Comment:
please add https://www.yggtorrent.is/ to the legacylinks section so that users' configs get auto corrected. thanks.

Reply:
My bad, I had URL without subdomain in cache of my browser.
Fixed commit.